mkosi/__init__.py

```diff
@@ -1913,11 +1911,11 @@ def install_fedora(args: CommandLineArguments, root: str, do_run_build_script: b
         updates_url = f"baseurl={args.mirror}/updates/{args.release}/Everything/$basearch/"
     else:
         release_url = (
-            f"metalink=https://mirrors.fedoraproject.org/metalink?" + f"repo=fedora-{args.release}&arch=$basearch"
+            "metalink=https://mirrors.fedoraproject.org/metalink?" f"repo=fedora-{args.release}&arch=$basearch"
```

Comment:
Yeah, that's because it does put things on a single line as long as they fit. 

I do see your point about readability, but I'm not a fan of the needless implicit concatenation, when they're on the same line. You could just switch of black on these lines with a format comment.